etc/schema.json

```diff
@@ -2014,6 +2014,9 @@
                 "status": {
                     "type": "integer"
                 },
+                "status_str": {
```

Comment:
We have no occurrences of `_str`, but do have one as `_string`, so maybe we should stick with that.

Also, please add a description.  I wonder if that description should also include the rule keyword for matching? Or if not applicable, even state that somehow?

Reply:
guess a `http.stat_code.raw`? or `http.stat.raw`

Reply:
I guess I will go with `status_string` instead of opening a new json object